gene_finder.py

```diff
@@ -45,9 +50,16 @@ def get_reverse_complement(dna):
     >>> get_reverse_complement("CCGCGTTCA")
     'TGAACGCGG'
     """
-    # TODO: implement this
-    pass
+    reverse = ''
+    i = 0
+    length = len(dna)
 
+    while i < length:
+        letter = dna[length - 1 -i] #moves backwards along the string
+        pair = get_complement(letter) #finds complement
+        reverse = reverse + pair
+        i += 1
+    return reverse
```

Comment:
fancy one line code (pythonic stylistic suggestion) : 
`return [get_complement(c) for c in dna[::-1]]`